fsfreeze.go

```diff
@@ -0,0 +1,150 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"syscall"
+)
+
+// from linux/fs.h
+const FIFREEZE = 0xC0045877
+const FITHAW = 0xC0045878
```

Comment:
I know this is a package main so it doesn't strictly matter much, but Go style is to not export/capitalize these unless you want it part of your public API. (https://go.dev/wiki/CodeReviewComments#mixed-caps)

This is more of an implementation detail so I'd unexport these, maybe prefixing them with `ioctl`